src/imu/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.5)
+project(imu)
+
+add_definitions(-D_USE_MATH_DEFINES)
+find_package(iDynTree REQUIRED)
+find_package(Eigen3 CONFIG REQUIRED)
```

Comment:
Adding these two additional dependencies has to be addressed carefully, maybe we can ad an option like `ICUB_TESTS_USES_IDYNTREE` and make `add_subdirectory` only if set to ON

Reply:
Ok, could we do something like this?

https://github.com/robotology/icub-tests/blob/1ed832991a878aad7507dac37e276ff9f0a60d6d/CMakeLists.txt#L73-L77

Reply:
Exactly but with idyntree, thanks!

Reply:
Here it is: https://github.com/robotology/icub-tests/pull/65/commits/0e1a3447d7b8e175afd56449f6895cb9d9d67a82

I removed the Eigen3 dependency since I think was a leftover from the very first implementation, it's not needed anymore

Reply:
I tried to compile after this change from a clear build and I didn't retrieve any error like the one mentioned in the issue you linked. Maybe it is due to the fact that I didn't link iDynTree libraries as PRIVATE. @traversaro could say more about

Reply:
I think that the reason is that you are compiling it dynamic, but I don't think that rtf supports static libraries since it loads plugins runtimes, then we can forget about it I think

Reply:
Yes, furthermore it is actually an iDynTree bug, so if we ever need that we can just fix the problem in iDynTree.